api_app/utility.py

```diff
@@ -0,0 +1,191 @@
+import logging
+from typing import Dict, List, Tuple
+from celery import uuid
+from django.conf import settings
+from api_app.connectors_manager.dataclasses import ConnectorConfig
+from api_app.helpers import get_now
+
+from api_app.analyzers_manager.dataclasses import AnalyzerConfig
+from api_app.exceptions import AlreadyFailedJobException, NotRunnableAnalyzer
+from api_app.models import Job
+from intel_owl.consts import DEFAULT_QUEUE
+from celery.canvas import Signature
+
+
+logger = logging.getLogger(__name__)
+
+def runnable_analyzers(
+    analyzers_to_execute: List[str]
+) -> List[str]:
+    analyzer_dataclass = AnalyzerConfig.all()
+    return [analyzer for analyzer in analyzers_to_execute if analyzer_dataclass.get(analyzer)]
+
```

Comment:
I **really** hate having an `utilities.py`. Having this file means that every function that you do not where to put, will be put there. For example this method should be a `@classmethod` of the `AnalyzerConfig` class

Reply:
Thank you for pointing this out. I appreciate you helping me clean the code up.

Reply:
Dw man. I just had some bad experiences with utilities files that grows exponentially and sorting them out is a huge pain in the ass that I would gladly not doing again